### PR TITLE
Fix building of kdbg

### PR DESCRIPTION
**Pull request by uleysky**
Add KDELibs4Support in CMakeLists.txt (i18n via klocale.h need this).
Remove unnecessary include from brkpt.cpp (this include removed in the recent versions of kdeframeworks).

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,6 +18,7 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
     IconThemes
     XmlGui
     WindowSystem
+    KDELibs4Support
```
Comment (j6t) [inline CMakeLists.txt]: Why is this needed? The goal is to have no KDE4 dependencies. Please mention the system where the build fails and the error message in the commit message. (This helps communicate *why* this change is good.)

File: kdbg/brkpt.cpp [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
 #include <klocale.h>			/* i18n */
 #include <kiconloader.h>
-#include <ksimpleconfig.h>
```
Comment (j6t) [inline kdbg/brkpt.cpp]: This looks like an independent change. Can you please split it off into a separate commit? It would be nice to know what "recent" KDE frameworks means. (I guess it is all of KF5.)

File: CMakeLists.txt (hunk 1 above)
Comment (uleysky) [reply CMakeLists.txt]: Oh, I am sorry. I was glad to see kdbg ported to KF5 and hurried too much )
System is unstable gentoo with kdeframeworks 5.37.0. First error was
/tmp/build/kdbg/kdbg/pgmargs.cpp:8:34: fatal error: klocale.h: No such file or directory
The file klocale.h can be found in kdelibs4support. But you are right, this is not good fix. Using klocalizedstring.h instead of klocale.h will be better.

**Comment by j6t** [comment]
Superceded by pull request #2
